src/config/Configuration.tsx

```diff
@@ -19,5 +19,5 @@ export default class Configuration {
     { subdomain: 'slf', name: 'SAP Labs France (prod)' },
     { subdomain: 'slfcah', name: 'SAP Labs France (Charge@Home) (prod)' },
   ];
-  public static isServerLocalePreferred: boolean = false;
+  public static isServerLocalePreferred: boolean = true;
```

Comment:
Not sure it's a good idea to override the phone default locale by the locale defined in server DB.
For testing purpose, that's fine but the end user expect to have the app in the language defined on his phone.

Reply:
This is the behavior in the front-end, I told Marian to do so.